docs/development.md

```diff
@@ -0,0 +1,134 @@
+# Development Environment setup
+
+## Pre-reqs
+
+The below instructions assume Argo CloudOps is on your local OSX system
+with Docker Desktop managing resource in AWS (region us-west-2) with credentials provided by Vault.
+
+* Install [Docker Desktop](https://www.docker.com/products/docker-desktop), ensure kubernetes is running.
+
+* Install [Argo Workflows](https://argoproj.github.io/argo-workflows/installation/)
+
+* Install **Argo CLI** `brew install argo`
+
+* Install **GoLang** `brew install golang`
+
+* Install GoLint `go get -u golang.org/x/lint/golint` and ensure `$GOPATH` is in your `$PATH`.
+
+* Install [PostgreSQL](https://www.postgresql.org/download/)
```

Comment:
When I installed PostgreSQL using the website link above the quickstart script fails telling me to install PostgreSQL using `brew install postgresql` should probably just specify that instead of linking to the website.